src/App.js

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
+// import logo from './logo.svg';
+import {
+  BrowserRouter as Router, Route, Routes, Link,
+} from 'react-router-dom';
 import './App.css';
+import CreatorPage from './Pages/CreatorPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <div>Insert your component here!</div>
-      </header>
-    </div>
+    <Router>
```

Comment:
The Router in the main branch is moved to a higher-level component, so you don't need it here! Instead delete everything but the Routes section and merge in main to fix any conflicts before wrapping up the pr!